editor/animation_track_editor.cpp

```diff
@@ -3772,6 +3772,10 @@ void AnimationTrackEditor::_add_method_key(const String &p_method) {
 
 	List<MethodInfo> minfo;
 	base->get_method_list(&minfo);
+	Ref<Script> script(base->get_script());
+	if (script.is_valid()) {
+		script.ptr()->get_script_method_list(&minfo);
```

Comment:
This should happen automatically already if you check object.cpp, I have the feeling this is a bug

Reply:
May have to do with PlaceholderScriptInstance not being fed with methods when created, or not referencing the ones in Script, this may need changing there